### PR TITLE
USWDS - update v2 readme for npm

### PR DESCRIPTION
**Pull request by amyleadem**
## Description

Update the README for the `uswds` npm package to point users to the new `@uswds/uswds` package.  

Closes https://github.com/uswds/uswds/issues/4843

Preview link: [README in Github](https://github.com/uswds/uswds/pull/4846/files?short_path=b335630#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5)

### Problem
Users should be able to quickly and easily find the most recent version of USWDS to install. Currently, when you search for "USWDS npm" in Google or "uswds" on the [npmjs.com](https://www.npmjs.com/search?q=uswds) website, you are directed first to the `uswds` package instead of the current `@uswds/uswds` package. Once on the [`uswds` package page](https://www.npmjs.com/package/uswds), there is nothing to indicate that USWDS 3.0 or [@uswds/uswds](https://www.npmjs.com/package/@uswds/uswds) exists, so users can be left thinking that 2.13.3 is the most recent version. 

### Solution
Update the README for v2 to alert users that USWDS 3.0 exists, and that they can install it under the new scoped package at @uswds/uswds.

_Please note, this will update only the v2 readme that is pulled into the the `uswds` npm package page. This will not affect the v3 readme or the `@uswds/uswds` page_.

## Additional information

Before you hit Submit, make sure you’ve done whichever of these applies to you:

- [ ] Follow the [18F Front End Coding Style Guide](https://pages.18f.gov/frontend/) and [Accessibility Guide](https://pages.18f.gov/accessibility/checklist/).
- [x] Run `npm test` and make sure the tests for the files you have changed have passed.
- [ ] Run your code through [HTML_CodeSniffer](http://squizlabs.github.io/HTML_CodeSniffer/) and make sure it’s error free.
- [x] Title your pull request using this format: [Website] - [UI component]: Brief statement describing what this pull request solves.

### CODE REVIEW

**Review by bonnieAcameron — APPROVED** [review]
Checked for:

- [x] Typos
- [x] Formatting
- [x] Information gaps
- [x] Conflicting information

File: README.md [hunk 1]
```diff
@@ -2,19 +2,34 @@
 
 [![CircleCI Build Status](https://img.shields.io/circleci/build/gh/uswds/uswds/develop?style=for-the-badge&logo=circleci)](https://circleci.com/gh/uswds/uswds/tree/develop) ![Snyk vulnerabilities](https://img.shields.io/snyk/vulnerabilities/npm/uswds?style=for-the-badge) [![npm Version](https://img.shields.io/npm/v/uswds?style=for-the-badge)](https://www.npmjs.com/package/uswds) [![npm Downloads](https://img.shields.io/npm/dt/uswds?style=for-the-badge)](https://www.npmjs.com/package/uswds) [![GitHub issues](https://img.shields.io/github/issues/uswds/uswds?style=for-the-badge&logo=github)](https://github.com/uswds/uswds/issues) [![code style: prettier](https://img.shields.io/badge/code_style-prettier-ff69b4?style=for-the-badge)](https://github.com/prettier/prettier)
 
+## :mega: **USWDS 3.0 is now live** :mega:
+
+Starting with version 3.0, the latest updates to USWDS are now published to the [@uswds/uswds](https://www.npmjs.com/package/@uswds/uswds) package.
```
Comment (thisisdano) [inline README.md:7]: Rewritten with active voice and a little more detail

```suggestion
Starting with USWDS 3.0, we're publishing USWDS as a scoped package at [@uswds/uswds].(https://www.npmjs.com/package/@uswds/uswds).
```

File: README.md [hunk 2]
```diff
@@ -2,19 +2,34 @@
...
+
+This unscoped `uswds` package will remain active for legacy use.
```
Comment (thisisdano) [inline README.md:9]:
```suggestion
This unscoped `uswds` package will remain active for any updates to USWDS 2.x.
```

**Review by thisisdano — CHANGES_REQUESTED** [review]

**Comment by amyleadem** [comment]
@thisisdano Thanks for the review. I have committed your requested changes.

Now that I am revisiting, I do wonder if we should help users find the uswds npm page from the @uswds/uswds page as well. Maybe something more subtle in the [long-term support of 2.x section](https://www.npmjs.com/package/@uswds/uswds#user-content-long-term-support-of-v2x)? Let me know if you are interested in something like that and I can open up an issue for it.

**Comment by thisisdano** [comment]
Yes, let's do that. I'm going to merge this and we can make a new issue around adding the old package location to the long-term support section

**Pull request merged**